ut/s3_put_bucket_replication_action_test.cc

```diff
@@ -456,3 +464,49 @@ TEST_F(S3PutBucketReplicationActionTest, SendResponseToClientInternalError) {
   EXPECT_CALL(*request_mock, send_response(500, _)).Times(AtLeast(1));
   action_under_test_ptr->send_response_to_s3_client();
 }
+
+// The source and destination buckets cannot be the same
+TEST_F(S3PutBucketReplicationActionTest, ValidateDestinationIsNotSource) {
+
+  MockReplicationConfigStr =
```

Comment:
can we write this string in following way -
MockReplicationConfigStr =
 "	<ReplicationConfiguration 													"
 " 	xmlns=\"http://s3.amazonaws.com/doc/2006-03-01/\">							"
 "	<Role>role-string</Role> 													"
 "  <Rule>																	"
 "  <Status>Enabled</Status>													"
 "	<Priority>1</Priority> 														"
 "   <ID>Rule-1</ID>															"
 "	<DeleteMarkerReplication><Status>Disabled</Status></DeleteMarkerReplication>        "
 "	<Destination><Bucket>seagatebucket</Bucket></Destination>			               "
 "	<Filter><And><Prefix/>			                                                                               "
 "	<Tag><Key>key12</Key><Value>vv</Value></Tag>								"
 "	<Tag><Key>key2</Key><Value>value2</Value></Tag>							"
 "	</And></Filter></Rule>													"
 "	<Rule>																	"
 "	<Status>Enabled</Status>													"
 " 	<Priority>2</Priority>														"
 "	<ID>Rule-2</ID>															"
 "	<DeleteMarkerReplication><Status>Disabled</Status></DeleteMarkerReplication>         "										
 "	<Destination><Bucket>seagatebucket</Bucket></Destination>					"
 "	<Filter><And><Prefix/>													"
 "	<Tag><Key>key12</Key><Value>vv</Value></Tag>								"
 "	<Tag><Key>key2</Key><Value>value2</Value></Tag>							"
 "	</And></Filter></Rule>													"
 "	</ReplicationConfiguration>													";